pagerduty/event_orchestration_path_test.go

```diff
@@ -494,6 +494,64 @@ func TestEventOrchestrationPathGlobalUpdate(t *testing.T) {
 	}
 }
 
+func TestEventOrchestrationPathEscalationPolicyUpdate(t *testing.T) {
+	setup()
+	defer teardown()
+	input := &EventOrchestrationPath{
+		Type: "router",
+		Parent: &EventOrchestrationPathReference{
+			ID:   "E-ORC-1",
+			Self: "https://api.pagerduty.com/event_orchestrations/E-ORC-1",
+			Type: "event_orchestration_reference",
+		},
+	}
+
+	var url = fmt.Sprintf("%s/E-ORC-1/router", eventOrchestrationBaseUrl)
```

Comment:
I left the [comment](https://github.com/alexzakabluk/go-pagerduty/pull/3#discussion_r1664120129) in the original thread but we should probably use something other than `router` here, since routers don't support this actions. Just so the test data looks realistic.

Reply:
Oh my gosh. Copypasta error

Reply:
oh and to address the statement, I added a new test because the size of some of these makes it a little hard to reason about them (and also I wanted to familiarize myself with these go tests)